src/Microsoft.AspNet.Diagnostics/WelcomePage/Views/WelcomePage.cs

```diff
@@ -6,6 +6,12 @@ namespace Microsoft.AspNet.Diagnostics.Views
 #line default
 #line hidden
     ;
+#line 2 "WelcomePage.cshtml"
```

Comment:
I would definitely support that change if it weren't for the fact that they aren't built when you just run from VS, only when you build.cmd. Not including them would therefore confuse/mess up people who haven't run build.cmd.